pattern-library/sass/patterns/_icons.scss

```diff
@@ -1,486 +0,0 @@
-// ------------------------------
-// edX Pattern Library: Components - Icons
-
-// About: Contains base styling for icons, which are currently managed through IcoMoon (https://icomoon.io/) and rendered with the support and accesibility mindfullness of A Font Garde (https://github.com/filamentgroup/a-font-garde).
-
-// NOTE: Read more at https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Icons
-
-// #SETTINGS
-// #GLOBAL
-// #ICONS
-// #FALLBACKS
-
-
-// ----------------------------
-// #SETTINGS
-// ----------------------------
-@import 'icons-afontgarde';
-
-$edx-icons:
-angle-down
-angle-left
-angle-right
-angle-up
-arrow-circle-down
-arrow-down
-arrow-up
-book
-bullhorn
-calendar
-camera
-caret-down
-caret-left
-caret-right
-caret-up
-check
-checkbox-checked
-checkbox-checked-dark
-checkbox-unchecked
-chevron-left
-chevron-right
-circle
-circle-light
-clock
-close
-close-circle
-close-circle-light
-closed-captions
-cog
-comment
-comments
-copy
-edit
-ellipsis
-email
-email-light
-error
-exit-fullscreen
-external-link
-facebook-square
-fast-backward
-fast-forward
-file
-file-light
-filter
-fullscreen
-flag
-help
-hide
-home
-information
-link
-linkedin-square
-list
-list-alt
-lock
-menu
-minus
-money
-pause
-pencil
-play
-plus
-print
-quote-left
-radio-checked
-radio-unchecked
-refresh
-reply
-search
-search-minus
-search-plus
-share
-shopping-cart
-show
-sitemap
-sort
-spinner
-square
-square-light
-star
-stop
-table
-thumb-tack
-trash
-twitter-square
-undo
-unlink
-unlock
-upload
-user
-video-camera
-volume-down
-volume-off
-volume-up
-warning;
-
-// Useful synonyms
-$edx-icon-synonyms: (
-    bars: menu,
-    delete: trash,
-    gear: cog,
-    hidden: hide,
-    reorder: menu,
-    remove: close,
-    settings: cog,
-    rotate-left: undo,
-    unsorted: sort,
-    visible: show
-);
-
-// ----------------------------
-// #GLOBAL
-// ----------------------------
-.icon {
```

Comment:
We probably want to keep this part, but it may be able to be moved into another partial.